frontend/src/metabase/dashboard/hoc/types.ts

```diff
@@ -1,46 +1,14 @@
 import type { Location } from "history";
-import type { LocationAction } from "react-router-redux";
 
-import type { EmbeddingDisplayOptions } from "metabase/public/lib/types";
+import type { DashboardDisplayOptionControls } from "metabase/dashboard/hoc/controls/use-dashboard-display-options";
 import type { DashboardId } from "metabase-types/api";
 
-import type { FetchDashboardResult } from "../types";
-
 // passed via ...this.props
 export type DashboardControlsProps = {
   location: Location;
-  fetchDashboard: (opts: {
```

Comment:
As mentioned above, we don't need to pass these `fetch` functions into `DashboardControls` anymore